static/css/master.css

```diff
@@ -1973,11 +1973,11 @@ div.Tools table td.vendor {padding:3px 10px 0 42px;}
 div.Tools table td.price {padding:3px 0 0 0;text-align:right;}
 div.Tools table.buy-options-table {
     border-spacing: 5px;
-    margin-left: 45px;
-    width: 200px;
+    margin-left: 0px;
+    width: 150px;
```

Comment:
I think we want to increase this to 175px. On the production site, we have an API key (which the dev clients may not have) which also fetches the price. When this happens, 150px is too small to contain the affiliate name and the price. e.g. https://dev.openlibrary.org/books/OL7389275M/Great_Physicists

Reply:
I had checked this manually adjusting the value. Give me some time to upload the relevant pic.

Reply:
It may also be slightly different on the /works versus the /books page(s)